src/scenario.jl

```diff
@@ -671,19 +689,20 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
             # Log site ranks
             # First col only holds site index ids so skip (with 2:end)
             site_ranks[tstep, rankings[:, 1], :] = rankings[:, 2:end]
-        elseif seed_corals && (in_seed_years || in_shade_years)
+        elseif seed_corals && (in_seed_years || in_fog_years)
```

Comment:
Hang on - doesn't this mean unguided selection only occurs when there is something to seed?

What if `fogging > 0`?

Reply:
I think this should be changed to:

```
# somewhere above...
apply_fogging = fogging > 0.0

        elseif (seed_corals && in_seed_years) || (apply_fogging && in_fog_years)
```

Reply:
You're right!